app/src/main/java/com/example/encoratask/model/Character.kt

```diff
@@ -0,0 +1,46 @@
+package com.example.encoratask.model
+
+import org.json.JSONObject
+import java.io.Serializable
+
+/**
+ * Created by Erick Sanchez
+ * Revision 1 - 09/06/21
+ */
+class Character:Serializable {
+
+    var id:Int = 0
+    var name:String = ""
+    var status = ""
+    var species = ""
+    var gender = ""
+    var image:String = ""
+
+    constructor(obj:JSONObject) {
+
+        this.id = obj.optInt("id")
+        this.name = obj.optString("name")
+        this.status = obj.optString("status")
+        this.species = obj.optString("species")
+        this.gender = obj.optString("gender")
+        this.image = obj.optString("image")
+    }
+
+    companion object {
+
+        fun parse(response:String) : ArrayList<Character> {
+
+            var characters = ArrayList<Character>()
+            val obj = JSONObject(response)
+            val results = obj.optJSONArray("results")
```

Comment:
Same stuff with the name "results". It can be moved to a constants place

Reply:
mmmm, how? I dont understand =(

Reply:
Hardcoding the names of the JSON attributes (such as "results") might be a bummer to keep those names al through your code. A good practice is to keep all this names in a constants block (companion or a separated Constants file). If somehow the server changes the names you can go directly and change the constant value instead of look for it in your code

Reply:
Ok bro, I understand, thanks bro